packages/teleport-repl-component/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@teleporthq/teleport-repl-component",
+  "version": "0.8.1",
+  "description": "Repl editor component",
+  "author": "teleportHQ",
+  "license": "MIT",
+  "homepage": "https://teleporthq.io/",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/teleporthq/teleport-repl-component.git"
+  },
+  "bugs": {
+    "url": "https://github.com/teleporthq/teleport-repl-component/issues"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "es2015": "dist/esm/index.mjs",
+  "es2017": "dist/esm/index.mjs",
+  "types": "dist/types/index.d.ts",
+  "collection": "dist/collection/collection-manifest.json",
+  "collection:main": "dist/collection/index.js",
+  "unpkg": "dist/repl-component/repl-component.js",
+  "files": [
+    "dist/",
+    "loader/"
+  ],
+  "scripts": {
+    "build": "stencil build --docs",
+    "start": "stencil build --dev --watch --serve"
+  },
+  "devDependencies": {
+    "@stencil/core": "^1.0.7",
+    "rollup-plugin-node-builtins": "^2.1.2",
+    "rollup-plugin-node-globals": "^1.4.0"
+  },
+  "dependencies": {
+    "@teleporthq/teleport-component-generator-react": "^0.7.1",
+    "@teleporthq/teleport-component-generator-vue": "^0.7.0",
+    "codeflask": "^1.4.1"
```

Comment:
Hey, I think we should move ``@stencil/core`` and add ``prismjs`` to the **dependencies**

Reply:
prismjs yes, but stencil is a dev-dependency as it is not part of the bundle in the end

Reply:
actually, you are right, going to change it

Reply:
I see few ```rollup`` plugins which are being used in the ``stencil-config`` are also missing in package.json. Please cross check that too 👍

Reply:
it's a bit strange that these are required in `dependencies` and not `devDependencies`. after all they are used as `typescript` and `ts-node` in configuration file. the linter here is looking blindly at all the files, I will check if we can strictly analyze the `src` folders in all packages